examples/ResampleAxisAligned.ipynb

```diff
@@ -0,0 +1,166 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.        # direct_transform.ApplyToImageMetadata(labels)    
</pre><p>Do these require itk-5.3? If so, could a comment please be added accordingly?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/KitwareMedical/HASI/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
I added a comment there and force-pushed.